components/switch/package.json

```diff
@@ -16,11 +16,11 @@
     "build": "gulp"
   },
   "peerDependencies": {
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.3"
```

Comment:
We can pull in version `1.0.7` now.

Reply:
Sure!.
This is going to be a maintenance thing for a while to keep pulling the latest tokens and verifying them via VRT.